src/templates/coffee.njk

```diff
@@ -14,8 +14,9 @@
             <div class="col">
                 <div class="card bg-light pb-4">
                     <h3 class="card-title mt-3 ms-4">Quejina
-                        <span class="d-block d-sm-inline d-md-block d-xl-inline">(Guatemala)</span></h3>
-                    <img src="/img/smcr-quejina.jpg" class="border-top" alt=""/>
+                        <span class="d-block d-sm-inline d-md-block d-xl-inline">(Guatemala)</span>
+                    </h3>
+                    <img src="/img/smcr-quejina.jpg" class="border-top card-img-top" alt="" width="480" height="480"/>
```

Comment:
dobrý i když bych dal opačně pořadí tříd
tedy první komponenta, druhá utilita
ale není to zásadní

Reply:
ok